.ci/gen-workflow-files.nu

```diff
@@ -103,22 +106,22 @@ for system in ($targets | columns) {
   for derivation in $derivations {
 
     # job_id for GH-Actions
-    let id = ( job-id $system $derivation )
+    let id = ( job-id $derivation )
 
     # name displayed
-    let name = $"($system).($derivation)"
+    let name = ( job-id $derivation )
 
     # collection of dependencies
     # TODO currently only considers dependencies on the same $system
     let needs = ($derivations
-      | filter {|it| $it != $derivation and $it != "default" } # filter out self and default
+      | filter {|it| $it != $derivation and $it != $".#packages.($system).default" } # filter out self and default
```

Comment:
We can remove the default filtering since 3eafb8e5ca1d30370c960d812fb183187d9d1850 is a better solution for the original problem that made me add this.